Dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG PYTHON_VERSION=3.7
+
+FROM python:${PYTHON_VERSION}
+
+RUN apt-get update \
+    && apt-get -y install openmpi-bin libopenmpi-dev \
+    && apt-get -y install swig  \
+    && rm -rf /var/lib/apt/lists/* \
+    && pip install --no-cache-dir -U 'pip<20' \
```

Comment:
Do we have any reason to restrict the pip version?

Reply:
Good catch!
We saw a problem at that point (see: https://github.com/optuna/optuna/pull/911) and now no need to use old pip.